README.md

```diff
@@ -1,3 +1,153 @@
+- **Database**: PostgreSQL, Prisma as ORM
+- **Authentication**: JSON Web Tokens (JWT)
+- **Logging**: Winston for Node.js
+- **Web Sockets for dynamic chatrooms**
+
+## Installation
+
+### Prerequisites
+
+- Node.js and npm installed
+- Expo CLI installed
+- PostgreSQL installed and running
+
+### Backend Setup
+
+1. Clone the repository:
+    ```bash
+    git clone https://github.com/KipaLogix/SporTogether.git
+    cd sportogether/backend
+    ```
+
+2. Install dependencies:
+    ```bash
+    npm install
+    ```
+
+3. Create a `.env` file in the `backend` directory and add your environment variables:
+    ```env
+    DATABASE_URL="postgresql://postgres:postgres@localhost:5432/SporTogether?schema=public"
+    PORT=3001
+    SECRET=q3049h8gfhq30n9fa3w90vnh08hN89029329@J!E$
+    ```
+4. Initialize Prisma and run migrations:
+   ```bash
+    npx prisma init && npx prisma migrate dev
+    ```
+
+5. Start the backend server:
+    ```bash
+    npm start
+    ```
+
+### Frontend Setup
+
+1. Navigate to the `frontend` directory:
+    ```bash
+    cd ../frontend/ExpoClient
+    ```
+
+2. Install dependencies:
+    ```bash
+    npm install
+    ```
+
+3. Start the Expo development server:
+    ```bash
+    npx expo
+    ```
+
+## Usage
+
+1. Ensure the backend server is running.
+2. Start the Expo development server.
+3. Use the Expo client app on your mobile device to scan the QR code displayed in the terminal.
+4. Sign up or log in to start creating and joining sports events, chatting and having a great time.
+
+## API Endpoints
+
+### User Authentication
+
+- **POST /api/users/register**: Register a new user.
+- **POST /api/users/login**: Log in an existing user.
+
+### Events
+
+- **GET /api/events**: Get a list of all events by location and area.
+- **POST /api/events**: Create a new event.
+- **GET /api/events/:id**: Get details of a specific event.
+- **GET /api/events/myEvents/:userid**: Get a list of all events of a user.
+- **POST /api/events/join**: Join an event.
+- **POST /api/events/leave**: Leave an event.
+- **DELETE /api/events/cancel**: Cancel an event.
+
+### Users
+
+- **POST /api/users/**: Create a new user.
+- **GET /api/users/:id**: Get user profile details.
+- **PUT /api/users/:id**: Update user profile details.
+- **DELETE /api/users/:id**: Delete user.
+
+### Sports
+
+- **GET /api/sports**: Get a list of all sports.
+
+### Messages
+
+- **POST /api/messages/**: Create a new message.
+- **GET /api/messages/:eventId**: Get all messages corresponding to an event.
+
+## Contributing
+
+Contributions are welcome! Please follow these steps:
+
+1. Fork the repository.
+2. Create a new branch (`git checkout -b feature/your-feature`).
+3. Commit your changes (`git commit -am 'Add some feature'`).
+4. Push to the branch (`git push origin feature/your-feature`).
+5. Create a new Pull Request.
+
+## Contact
+
+If you have any questions or suggestions, please feel free to contact us at [domibacsro@gmail.com](mailto:domibacsro@gmail.com).
```

Comment:
+ [amarietiberiu@yahoo.com](mailto:amarietiberiu@yahoo.com) 
( I'm also open for feedback 😁 )